apps/org/urls.py

```diff
@@ -21,4 +22,11 @@
     url(r'^organization/(?P<pk>[0-9]+)/delete/$',
         DeleteOrganizationView.as_view(),
         name='organization-delete'),
+    url(r'^org-member-api$',
+        LocalUserAPI.as_view(),
+        name='org-member-api'),
+    url(r'^search$',
```

Comment:
could the search behavior that's under `org-member-api` be placed under the `search` view as a `post`? I think that cleans up the naming a bit.